mautic/mautic
# Only push companies for integration syncs if push is enabled

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Pulling companies only happened if the `Pull contacts from integration` feature was checked. But we pushed companies regardless. This changes it so that companies will only push if `Push contacts from integration` is enabled and it updates the language string to include companies. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Configure the SF CRM plugin
2. Check Contacts, Leads, and Accounts as objects
3. Check `Pull contacts from integration` but don't check `Push contacts to this integration`
4. Run the 'php app/console m:i:s -i Salesforce` command
5. Notice that it'll end in `Number of companies pushed...`

#### Steps to test this PR:
1. Repeat and this time companies should NOT be pushed
2. Check `Push contacts and/or companies to this integration` and run the command again and this time it should push

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
